recipes/openforcefield/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openforcefield" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cd4ca0ded60ebd97120523b85c2aa6e0ddbc400193e6bbf49aa52b82ccf9e026
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - smirnoff99frosst
+    - openforcefields
+    - openmm
+    - networkx
+    - rdkit
+    - ambertools >=20
+    # Serialization: Should these be optional?
+    - toml
+    - bson
+    - msgpack-python
+    - xmltodict
+    - pyyaml
+    - cairo >=1.16
```

Comment:
> Should these be optional?

Almost surely yes. We'll want to make sure those functions fail gracefully before removing them, though